debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/RecordsStreamProducer.java

```diff
@@ -539,9 +538,10 @@ private boolean hasMissingUntoastedColumns(Table table, List<ReplicationMessage.
                 .collect(Collectors.toList());
 
         // Compute list of table columns not present in the replication message
-        List<String> missingColumnNames = table.columnNames()
+        List<String> missingColumnNames = table.columns()
```

Comment:
@jpechane that's the only occurrence in the hot loop (=non tests, non DDL events, but DML event handling) where the mapping is needed. But I think it only happens rarely (in case of changed schema), so it should be fine. WDYT?